docs/src/modules/ROOT/pages/enterprise-edition/enterprise-edition.adoc

```diff
@@ -558,16 +546,11 @@ There are additional parameters you can supply to your `solverConfig.xml`:
 <solver xmlns="https://timefold.ai/xsd/solver" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
     xsi:schemaLocation="https://timefold.ai/xsd/solver https://timefold.ai/xsd/solver/solver.xsd">
   <moveThreadCount>4</moveThreadCount>
-  <moveThreadBufferSize>10</moveThreadBufferSize>
```

Comment:
Yep, this parameter has been always "a bit too much". Can we remove/deprecate it?

Reply:
I'm open to the conversation. Feel free to start it.